pom.xml

```diff
@@ -42,6 +42,11 @@
   </scm>
   
   <dependencies>
+      <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>jsch</artifactId>
+          <version>0.1.54.2</version>
+      </dependency>
       <dependency>
           <groupId>org.jenkins-ci.lib</groupId>
           <artifactId>cvsclient</artifactId>
```

Comment:
Would it it make sense to explicitly add an exclusion here? Otherwise we may still have an issue with bundling of the library dependencies after the patch

Reply:
Should not be necessary:

* `requireUpperBoundsDeps` already enforces that the desired version is selected. (For example, if you swap the order of the two explicit deps, the build fails.)
* I manually verified that `cvs.hpi` contained `jsch.jar` before but not after this patch.

Reply:
Will it also exclude transient deps in such case. E.g. if the previous jsch library includes foo.bar which is no longer bundled in 0.1.54

Reply:
It should, but anyway there are no such dependencies.

Reply:
Specifically `mvn:dependency-tree`:

```diff
--- before	2018-02-28 15:46:22.539439704 -0500
+++ after	2018-02-28 15:46:31.043479035 -0500
@@ -1,6 +1,11 @@
 [INFO] org.jenkins-ci.plugins:cvs:hpi:2.14-SNAPSHOT
+[INFO] +- org.jenkins-ci.plugins:jsch:jar:0.1.54.2:compile
+[INFO] |  +- com.jcraft:jsch:jar:0.1.54:compile
+[INFO] |  \- org.jenkins-ci.plugins:ssh-credentials:jar:1.13:compile
+[INFO] |     \- org.jenkins-ci.plugins:credentials:jar:2.1.0:compile
+[INFO] |        \- org.antlr:antlr4-runtime:jar:4.5:compile
+[INFO] |           \- org.abego.treelayout:org.abego.treelayout.core:jar:1.0.1:compile
 [INFO] +- org.jenkins-ci.lib:cvsclient:jar:71-jenkins-11:compile
-[INFO] |  \- com.jcraft:jsch:jar:0.1.50:compile
 [INFO] +- org.reflections:reflections:jar:0.9.9:test
 [INFO] |  +- com.google.guava:guava:jar:15.0:provided
 [INFO] |  \- org.javassist:javassist:jar:3.18.2-GA:test
```